var/spack/repos/builtin/packages/py-neurom/package.py

```diff
@@ -14,12 +14,8 @@ class PyNeurom(PythonPackage):
     url = "https://pypi.io/packages/source/n/neurom/neurom-2.2.1.tar.gz"
 
     version('develop', branch='master')
-    version('2.2.1',  sha256='72f6a552b53ced520a3ccb1a2986ff95ff91df99d9227213f671f87a8d8b6499')
-    version('2.1.2',  sha256='6ca24ce628cfa00ba63bc3bd362e9ad1cd337a4ad4d02a5a58ed82ec8e61ce97')
-    version('1.8.0',  sha256='d364d3b184bd96cbe5fa601ae24f6b7d431fa42de646e3011a33d56f3cfc247c')
-    version('1.7.0',  sha256='713d874538f1c566b57ab81e0558726fc6d4b7de91301a6be495776c55ac47f8')
-    version('1.6.0',  sha256='74759199c5392ae8e209f037a5046646d06ec1f77b1cd826afac71eeeca0f7ab')
-    version('1.5.0',  sha256='40a4362b58cbbbac769a1cef5b6e5e6ececbf4b538d81c0ed23fe421645aa3c4')
+    version('3.0.0',  sha256='05f5f5c4292dfa23f3319347cf2a7c147732ba0140b9db0bc94e535ac74be8da')
+    version('2.3.1',  sha256='d399b2ff22b4dfc1d9ac0f28be49d631a7a48a3fe0b9f13cb880ff60d0a1beba')
```

Comment:
AFAIK only the most recent version of a package is built in spack, unless:
- a different version is specified in `deploy/environments/applications.yaml`.
- or a previous version is required by other packages (in their latest version, or in a version explicitly specified)

This seems confirmed by the log https://bbpcode.epfl.ch/ci/blue/rest/organizations/jenkins/pipelines/hpc.spack-deployment.pull-requests/runs/2413/nodes/84/steps/89/log/?start=0 and also by the files in /gpfs/bbp.cscs.ch/apps/hpc/jenkins/pulls/1267

I see two alternatives:
- remove neurom 2.3.1 and the other old versions from the package definitions (they would not be built anyway). This would be ok if the versions in the previous archives are good enough, and we want to keep only the most up to date version in the current archive.
- specify the old versions in `deploy/environments/applications.yaml` in this PR. When it's merged, open a new PR to remove them. This would be ok if we want that these specific old versions will be available in `archive/2021-09` (because they are not available in the previous archives)

cc @matz-e for any correction or advice.

Reply:
Either suggestion works for me